fixpunkt/fp-newsletter/CVE-2022-47408.yaml

```diff
@@ -4,10 +4,7 @@ cve: CVE-2022-47408
 branches:
     master:
         time: 2022-12-13 10:48:00
-        versions: ['>=3.0.0', '<3.2.6']
-    2.x:
-        time: 2022-12-13 10:38:00
-        versions: ['>=2.0.0', '<2.1.2']
```

Comment:
This change looks wrong to me, as it removes 2.1.2 as a non-vulnerable version.

Reply:
Sorry, removed it by mistake.